src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2621,7 +2621,8 @@ void sub_6C94D8()
             type = _currentTrackPieceType;
             auto newCoords = sub_6C683D(
                 { _currentTrackBegin, static_cast<Direction>(direction) }, type, 0, nullptr,
-                _currentTrackSelectionFlags & TRACK_SELECTION_FLAG_ARROW ? 2 : 1);
+                _currentTrackSelectionFlags & TRACK_SELECTION_FLAG_ARROW ? TRACK_ELEMENT_SET_HIGHLIGHT_TRUE
```

Comment:
Can I suggest this parameter be split off into a const local variable? I really don't like ternaries in function calls, especially when they're spanning several lines.

Reply:
Neither do I ;-)